platform-sdk/swirlds-common/src/main/java/com/swirlds/common/wiring/wires/input/InputWire.java

```diff
@@ -29,19 +30,21 @@
  */
 public class InputWire<IN, OUT> {
 
-    private final TaskScheduler<OUT> taskScheduler;
+    private final TaskSchedulerInput<OUT> taskScheduler;
```

Comment:
Consider renaming member to `taskSchedulerInput`

Reply:
renamed